### DISCUSSION TITLE
Cache purpose

### DISCUSSION THREAD

**Discussion by rohan-kumar-nutanix** [aquasecurity/trivy#1866] [Q&A]
How exactly does trivy use the cache? Does it store all the vulnerabilities relating to an image in the cache, or something else? In the former, will I be required to clear the cache and rescan the image for getting the most updated vulnerabilities?

TIA

**Comment by ingentismg** [comment]
I was wondering the same. Can someone please add some details to the documentation.

**Comment by knqyf263** [comment]
SBOM is stored in cache so the next scan can skip analyzing installed packages. Detected vulnerabilities are always up-to-date.
